<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="page_0">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_newAccount">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Create a new account</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>521</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Welcome to the bank!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_Announcement">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser_InfoNewAccount">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>631</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Please state your name, surname, login and password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_newName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>New name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_newSurname">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>New surname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_newTempLogin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>New Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_newTempPassword">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>New password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_insertNewData">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_RepeatPassword">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>330</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_restatePassword">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Restate your password here</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_BackToMainMenuFromNewAccount">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Come back to main menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_restate">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Restate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_comeBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Come back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Please insert your login and password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_incorrectLogin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser_welcome">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_InsertMoney">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Insert money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_WithdrawMoney">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Withdraw money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_TransferMoney">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Transfer money</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_currentBalance">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_TakeOutLoan">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>150</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Take out a loan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_GiveBackMoney">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>200</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Give back money</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ComeBackMainMenu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Come back to main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_MoreOptions">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>300</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>More options</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser_MoneyGeneral">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_InsertAmount">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Insert amount of money here</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_confirmationMoney">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_insert">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_comeBackInsert">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>367</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Come back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_TransferToPerson_Name">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Name of the person to whom you would like to transfer the money</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_TransferToPerson_Surname">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Surname of the person to whom you would like to transfer the money</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser_welcomeBan">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Which user would you like to ban?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_enterUserNameBan">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Type in the user's name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_enterUserSurnameBan">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Type in the user's surname</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_BanUserInfo">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_comeBackBan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Come back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ban">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ban</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QTextBrowser" name="textBrowser_ListOfUsers">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>60</y>
        <width>256</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ComeBack_ListOfUsers">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>448</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Come back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_BanningTheUser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Ban a user</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_UnbanningTheUser">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Unban a user</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_changeNameAdmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the name of the user</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_changeSurnameAdmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the surname of the user</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_changeLoginAdmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the login of the user</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_changePasswordAdmin">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Change the password of the user</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
